README.md

```diff
@@ -2,9 +2,23 @@
 
 ## Development
 
-You'll want to make sure you have the Atlassian Dev environment and SDK installed (https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project)
+Development depends on a working Docker environment with docker-compose. The `Dockerfile` at the root of this repository defines a Debian base with the java 8 jdk and Atlassian Plugin SDK.
 
-To build (compile,test,jar) the plugin run: 
+You must first run docker-compose with `--build` to set up your build environment:
+
+```
+docker-compose up --build
+```
+Note: Subsequent `docker-compose up` shouldn't need `--build` unless you've changed `Dockerfile`.
+
+Once the `atlas-plugin-sdk` container has been created and running, you can attach to the running container with:
+```
+docker-compose exec atlas-plugin-sdk bash
+```
+
+From that shell it should be possible to run the following commands:
+
+To build (compile,test,jar) the plugin:
 
 ```
 atlas-package
```

Comment:
I'm not sure I follow? I thought you liked the idea of keeping the container stuff optional?